pom.xml

```diff
@@ -221,8 +223,23 @@
                     <dependencies>
                         <dependency>
                             <groupId>org.apache.maven.wagon</groupId>
-                            <artifactId>wagon-ssh</artifactId>
-                            <version>${wagon-ssh.version}</version>
+                            <artifactId>wagon-scm</artifactId>
```

Comment:
Is `wagon-scm` necessary for the scm publish plugin to do its work?  I don't see it documented at all on that plugin site. 

The dependencies below are all dependencies for `wagon-scm` so I am not sure if they are needed here.  They can also be put as extensions instead of dependencies if they are needed.

Reply:
extesion based on maven documentation was for maven 2.  maven 3 is in site directly.  Its very confusing as it seemed to be the other way some time ago.  Wagon-ssh is obsolete (in fact deleted now in recent release).  The other items in site (or should say based off other junk potentially in my main setup) resulted in scm not being same level so that just ensures it isn't messed up on resolution.  Do we need it all?  Possibly not, some of that issue my be unrelated and worth further review / cleanup.

Reply:
Looking at the plugin [pom](https://github.com/apache/maven-scm-publish-plugin/blob/ed970672a2878b6601c5a4771aa60bbf6352c041/pom.xml#L116-L125) it does include at least the api and scm-provider-gitexe as dependencies so those are probably fine to keep here at a higher level.  

I do not see wagon-scm or the plexus dependency so I think they can be deleted.

Reply:
Dug around in the deploy plugin. It does not use any of the scm/etc. stuff from POM.  It generates/stages the site and then has its own internal git implementation.   So I think we are safe with none of these dependencies, or explicitly specifying the two which are transitively imported anyway.  I actually think keeping `wagon-scm` might potentially cause problems as a competing implementation.

But I also know you've been testing all of this and trust your final judgment!

Reply:
wagon-scm and others will likely go away.  Since site won't be used to deploy, those are likey not necessary.  will look at that further after testing to streamline the setup.